botocore/docs/bcdoc/docstringparser.py

```diff
@@ -129,29 +135,32 @@ def __init__(self, tag, attrs=None, parent=None):
         self.attrs = attrs
         self.tag = tag
 
-    def write(self, doc):
+    def write(self, doc, next):
         self._write_start(doc)
         self._write_children(doc)
-        self._write_end(doc)
+        self._write_end(doc, next)
 
     def _write_start(self, doc):
         handler_name = 'start_%s' % self.tag
         if hasattr(doc.style, handler_name):
             getattr(doc.style, handler_name)(self.attrs)
 
-    def _write_end(self, doc):
+    def _write_end(self, doc, next):
```

Comment:
can you also rename `next` to something more detailed like `next_child` or something along those lines? `next` is a builtin function so we should avoid that namespace collision